fix(shared-data): Sync `innerLabwareGeometry` to latest hardware values

SyntaxColoring opened the pull request:
## Overview

Goes towards EXEC-1480.

## Changelog

**Done in this PR:**

* Update all labware definitions' `innerLabwareGeometry` to the latest values from the hardware spreadsheet.

  This was mostly done by [script](https://github.com/SyntaxColoring/labware-scripts/blob/8a963ecf9d099199a8e8017bf3e189e17366dc99/import_frusta_from_hw.py), but unfortunately it needed a lot of manual intervention. Check the commit messages for details.

   I intend and believe this sync to be exhaustive in the sense that it hits every labware named in the "Frustum Definitions" tab, and also some labware not explicitly named, like other tube racks that share the same tubes.

* Update the values in our height<->volume calculation tests to the latest values from the hardware spreadsheet.

* Add `innerLabwareGeometry` to `opentrons_96_aluminumblock_generic_pcr_strip_200ul`. It's in the spreadsheet but was missing from our JSON files. This addition strikes me as a little bit iffy because it's a generic labware, so how could we have detailed geometry for it? But we do already have `innerLabwareGeometry` in `opentrons_24_aluminumblock_generic_2ml_screwcap`, so I guess it's okay?

* For the files whose `innerLabwareGeometry` changed, fix up their labware `zDimension`s, their well `depth`s, and their well `z`s so everything still adds up. Favor the latest numbers from the hardware spreadsheet.

**Not done in this PR, but will be done in follow-ups:**

* Checking any labware's `xDimension`, `yDimension`, `well.x`, or `well.y` values.
* Checking `zDimension`, `well.depth`, or `well.z` for any labware whose `innerLabwareGeometry` did not change in this PR.
* Updating the Python Protocol API to load these new labware versions by default on new `apiLevel`s.

## Test Plan and Hands on Testing

* [x] Checked a `usascientific_12_reservoir_22ml` on a robot. The pipette definitely goes noticeably lower now. I trust that this is a good thing, given the hardware testing that the numbers in the spreadsheet have already gone through.

## Review requests

* Agreed with adding `innerLabwareGeometry` to `opentrons_96_aluminumblock_generic_pcr_strip_200ul`? Or do we want to go the other way, and remove `innerLabwareGeometry` from all generic labware?

* Double-check my manual changes:
  * Parts of https://github.com/Opentrons/opentrons/pull/18266/commits/6383f9da7e71c79252b8fedfd0f99fc273b2c94a, see the commit message
  * https://github.com/Opentrons/opentrons/pull/18266/commits/e6f44ee178ce9fbc427cb7380ed2dfd57beb2543
  * https://github.com/Opentrons/opentrons/pull/18266/commits/33c3da6301f779709ae83db3f1ac99bf69afb6d6
  * https://github.com/Opentrons/opentrons/pull/18266/commits/8aeef7bc2b71cd351df92bfe4e7e0c84e55c4da6
  * https://github.com/Opentrons/opentrons/pull/18266/commits/cf44a46525880ccafaa53d477c1fd1d53e964e6c

* Review the script (linked above), if you like.
  * And its results in https://github.com/Opentrons/opentrons/pull/18266/commits/6383f9da7e71c79252b8fedfd0f99fc273b2c94a

## Risk assessment

No risk to existing protocols, since I'm making these changes in new labware versions.

There is a risk of bad numbers getting lost in the noise. For example, I've seen x-dimensions and y-dimensions get swapped. We do have [tests](https://github.com/Opentrons/opentrons/blob/c24eb5d52f108360583be144a2a383151a02e5c0/shared-data/js/__tests__/labwareDefSchemaV2.test.ts#L231) that cover most of the mistakes that I've noticed so far, but I'm sure stuff can still sneak through.

SyntaxColoring committed:
Create drafts from the latest versions of each labware.

SyntaxColoring committed:
Sync innerWellGeometry from hardware spreadsheet.

Notes:

- Run import_frusta_from_hw.py.
- Manually fix up the output to reduce the diff. e.g. undo field reorderings, and undo changes like 0->0.0.
- The following labware erroneously have a `PATTERN QTY` in the spreadsheet, which will try to create `xCount` and `yCount` here. Undo that.
  - corning_12_wellplate_6.9ml_flat
  - corning_384_wellplate_112ul_flat
  - nest_12_reservoir_15ml
  - nest_96_wellplate_2ml_deep
  - opentrons_96_wellplate_200ul_pcr_full_skirt
- nest_1_reservoir_195ml has extra junk in the spreadsheet ("--> 0" where I think it should just be "0"), which the script chokes on. Just verify manually that it matches the spreadsheet.
- The following tube racks have heterogeneous wells and are split up in the spreadsheet, which we can't handle automatically. Handle them by doing hacky one-off temporary modifications to the script.
  - opentrons_10_tuberack_nest_4x50ml_6x15ml_conical
  - opentrons_10_tuberack_falcon_4x50ml_6x15ml_conical
- The script needed some help with opentrons_96_aluminumblock_generic_pcr_strip_200ul because it didn't have `innerLabwareGeometry` to begin with.
- The script chokes on opentrons_24_tuberack_generic_0.75ml_snapcap_acrylic, but it has empty data in the sheet and it's a retired labware, so just leave it unmodified.

SyntaxColoring committed:
Rename drafts to properly numbered versions.

Delete any drafts that we didn't modify.

SyntaxColoring committed:
Copy opentrons_24_tuberack_nest_0.5ml_screwcap to opentrons_24_aluminumblock_nest_0.5ml_screwcap.

Only the former is mentioned in the hardware spreadsheet, but the latter uses the same physical tubes and therefore ought to share the same geometry. Fortunately, we have tests to catch this.

SyntaxColoring committed:
Fix heights and elevations in usascientific_12_reservoir_22ml.

- Change zDimension to 44.2, which comes directly from the hardware spreadsheet.
- Change well bottom z to 2.45, which is chosen so that well.z + well.depth == labware.dimensions.zDimension. We trust well.depth and labware.dimensions.zDimension because they both come straight from the hardware spreadsheet.

SyntaxColoring committed:
Fix heights and elevations in agilent_1_reservoir_290ml.

- Change well depth to 39.23, which comes directly from the hardware spreadsheet.
- Change well bottom z to 4.81 so that well.z + well.depth == labware.dimensions.zDimension. zDimension already has a value that comes directly from the hardware spreadsheet.

SyntaxColoring committed:
Fix heights and elevations in corning_47_wellplate_1.6ml_flat.

- Change well depth to 17.399, which comes directly from the hardware spreadsheet.
- Change well bottom z to 2.621 so that well.depth + well.z == labware.dimensions.zDimension. zDimension already has a value that comes directly from the hardware spreadsheet.

SyntaxColoring committed:
Update quick-transfer constants.

caila-marashaj commented on shared-data/labware/definitions/2/agilent_1_reservoir_290ml/3.json:
```diff
@@ -0,0 +1,80 @@
+    "cuboidalWell": {
+      "sections": [
+        {
+          "shape": "cuboidal",
```
Is there any need for the names of sections to be uniform? This is fine with me if so because the name makes sense, but I'm just wondering if there's any json weirdness that will bite us in the but for not having a finite list of possible section names or anything like that

SyntaxColoring replied:
Spoke in-person!

SyntaxColoring committed:
Update test params.

These numbers come from the spreadsheet. The short story is that the existing numbers were stale and needed to be updated. Those updates are in the sheet now, and this commit copies them from the sheet.

caila-marashaj reviewed (approved):
Nice! Checked the measurements against what's on the spreadsheet and it looks good
